pages/sign-up.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import Image from 'next/image';
-import { Button, Input, MnavBar } from '@/components/index';
+import { Button, CheckboxGroup, CheckboxGroups, Input, MnavBar } from '@/components/index';
```

Comment:
CheckboxGroups is no longer used, so it is safe to remove here.

```suggestion
import { Button, CheckboxGroup, Input, MnavBar } from '@/components/index';
```

Reply:
It looks like you missed this comment 😸